File: public/benchmarks/fib/zig/fib.zig
const std = @import("std");
fn fib(n: i64) i64 {
    if (n < 2) return n;
    return fib(n - 1) + fib(n - 2);
}
pub fn main() void {
    const r = fib(37);
    std.debug.print("{d}\n", .{r});
}

File: public/benchmarks/fib/go/fib.go
package main
import "fmt"
func fib(n int) int64 {
    if n < 2 {
        return int64(n)
    }
    return fib(n-1) + fib(n-2)
}
func main() {
    fmt.Println(fib(37))
}
